pilot/lib/pilot/tester_manager.rb

```diff
@@ -67,7 +67,14 @@ def remove_tester(options)
         # If no groups are passed to options, remove the tester from the app-level,
         # otherwise remove the tester from the groups specified.
         if config[:groups].nil? && tester.kind_of?(Spaceship::Tunes::Tester::External)
-          test_flight_tester = Spaceship::TestFlight::Tester.find(app_id: app.apple_id, email: tester.email)
+          test_flight_testers = Spaceship::TestFlight::Tester.search(app_id: app.apple_id, text: tester.email)
```

Comment:
Yeah, I was relying on the fall through logic to catch matches that are > 1, but you make a good point. I just updated it so pass in the `is_email_exact_match` params as `true` to get this behavior.